docs/configuration.md

```diff
@@ -58,7 +58,17 @@ you wish to keep in git, but you wish to hide from from SeaGOAT.
     * `name`: Name of the embedding function to use.
     See [ChromaDB's docs for more](https://docs.trychroma.com/embeddings)
     * `arguments`: Arguments to pass to the embedding function.
-    For example, use `preferred_providers` to control the execution provider
+    * Example: if you wanted to use the `ONNXMiniLM_L6_V2` embedding model with TensorRT
```

Comment:
this way it will be formatted as a separate bullet point I think. you can run `poetry run mkdocs serve` to check how it would look

Reply:
how it looks now:

![2023-10-17-152054_1309x558_scrot](https://github.com/kantord/SeaGOAT/assets/3704904/38d2acfb-e333-4b7d-b5fb-eea011832aeb)

Reply:
resolved in commit 1bc60b2